### PR TITLE
feat(release): add release notes for kubera-1.12

### PR DESCRIPTION
**Pull request by AmitKumarDas**
Signed-off-by: AmitKumarDas <amit.das@mayadata.io>

### CODE REVIEW

**Review by harshshekhar15 — APPROVED** [review]
/lgtm

File: testplan/docs/kubera-e2e-release-1.12.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+### Rancher E2E Pipeline
+In addition to AWS and KONVOY, E2E uses Rancher as a Kubernetes platform to run end to end test cases. Rancher pipeline still continues to face a lot of issues. This has led to either its pipeline runs getting cancelled or marked as failed. In this release, it was mostly due to frequent reboots of underlying VMware VMs. As of today, these VMs have their motherboard replaced and things have started to look fine. Team will focus diligently to elevate Rancher as a stable pipeline in upcoming release.
```
Comment (prabhatkumarthakur) [inline testplan/docs/kubera-e2e-release-1.12.md:11]: it was mostly due to frequent reboots of Esx.... As of today, the motherboard of this ESX had been replaced.

**Review by bhaskarhc — APPROVED** [review]
LGTM

File: testplan/docs/kubera-e2e-release-1.12.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+### Gitlab cluster license renewal
```
Comment (rohan2794) [inline testplan/docs/kubera-e2e-release-1.12.md:13]: @AmitKumarDas 
can we update `Gitlab cluster license renewal`  to `GitLab  license renewal`

**Review by amitbhatt818 — APPROVED** [review]
/lgtm

File: testplan/docs/kubera-e2e-release-1.12.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+#### E to E team will continue to focus on the following
+- Develop or use declarative tools to:
+    - setup & dispose Kubernetes clusters
+    - exercise granular control to run tests, stages & pipelines
+    - write integration as well as e2e test cases
+    - find the root cause of errors in the pipelines
+    - to analyse current state of release from E2E metrics
+- Implement e2e test cases from users' view of the world
+  - i.e. validate & verify managing workloads on Kubera
+    - Run Day 0, Day 2 operations
```
Comment (gprasath) [inline testplan/docs/kubera-e2e-release-1.12.md:46]: should we include day1 as well?

**Review by rohan2794 — APPROVED** [review]
LGTM

**Pull request merged**